sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -788,7 +788,7 @@ public HttpPipeline getHttpPipeline() {
                         objectPagedResponse.getHeaders(),
                         convertedList,
                         null,
-                        ((PagedResponseBase) objectPagedResponse).getDeserializedHeaders());
```

Comment:
@azabbasi I saw that you made this cast be for PagedResponseBase, but I had to revert that change to make this function work. Is that okay?

Reply:
Just fyi - this is correct, the return type of the add model PL API is of type `Response`, so it needs to be cast to `ResponseBase`. I've had to make the same change in the samples PR as well #14781

Reply:
This is no longer required, we have switched the return type back to List and this is now outdated.